projects/aca-content/assets/i18n/en.json

```diff
@@ -259,7 +259,8 @@
             "LEAVE": "Leave Library",
             "EDIT_OFFLINE": "Edit Offline",
             "EDIT_OFFLINE_CANCEL": "Cancel Editing",
-            "CHANGE_ASPECT": "Edit Aspects"
+            "CHANGE_ASPECT": "Edit Aspects",
+            "CLOSE": "Close"
```

Comment:
Hmm do we need add this translation? Can't we reuse already existing translation for Close which is in ADF?
<img width="909" alt="image" src="https://github.com/Alfresco/alfresco-content-app/assets/115619721/85d199e5-0c21-48ab-8c22-8ef5685b3dbb">

Please try to pass this translation key for close from ADF to see if that works.